src/lava/magma/core/model/py/model.py

```diff
@@ -76,8 +76,8 @@ def join(self):
 
 
 class PyLoihiProcessModel(AbstractPyProcessModel):
-    def __init__(self):
-        super(PyLoihiProcessModel, self).__init__()
+    def __init__(self, proc_params: ty.Dict[str, ty.Any] = {}):
```

Comment:
No need to make this optional. If you initialize proc_params in AbstractProcess then there is always a proc_params the Builder can pass.

Reply:
Fixed it.